components/fxa-client/src/device.rs

```diff
@@ -197,7 +197,7 @@ impl FirefoxAccount {
 }
 
 /// Device configuration
-#[derive(Debug)]
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
It's needed because `DeviceConfig` is a field in the `Event::Initialize` and those need to be compared (maybe only for the tests though, I'm not sure).